src/test/scala/org/galaxio/gatling/kafka/examples/KafkaGatlingTest.scala

```diff
@@ -184,7 +184,7 @@ class KafkaGatlingTest extends Simulation {
   val scnAvro4s: ScenarioBuilder = scenario("Request Avro4s")
     .exec(
       kafka("Request Simple Avro4s")
-        .send(Ingredient("Cheese", 1d, 50d)),
+        .send[Ingredient](Ingredient("Cheese", 1d, 50d)),
```

Comment:
Type parameters add extra noise to the code, as you can see. It's okay in Scala because you can skip it and the compiler will derive it, but in Java, you should pass it everywhere.